scripts/test.sh

```diff
@@ -22,6 +22,7 @@ run () {
     yarn mocha $(find desktop/apps/*/test -name '*.coffee')
     yarn mocha $(find desktop/apps/*/test -name '*.js')
     yarn mocha $(find desktop/apps/*/**/*/test -name '*.coffee')
+    yarn mocha $(find desktop/apps/**/__tests__ -name '*.js')
```

Comment:
👍  I'm all for colocated tests. We've been colocating by app/component since the early days and I'm happy to see Jest popularize somewhat of a standard for this.